Sources/Containerization/SandboxContext/SandboxContext.proto

```diff
@@ -34,6 +34,8 @@ service SandboxContext {
   // Resize the tty of a given process. This will error if the process does
   // not have a pty allocated.
   rpc ResizeProcess(ResizeProcessRequest) returns (ResizeProcessResponse);
+  // Close IO for a given process.
+  rpc CloseProcessStdin(CloseProcessStdinRequest) returns (CloseProcessStdinResponse);
```

Comment:
Could this be a more generic API? Something like `CloseFD` and you provide it an fd number to close?

Reply:
the socket is different but should the client know fd 0,1,2?  If that would be too much, maybe a bool allowing any fd to be closed via an enum?  `stdin|out|err`

Reply:
We chatted offline, but this should be fine as is